rosco-web/config/rosco.yml

```diff
@@ -39,12 +39,13 @@ executionStatusToBakeResults:
 
 defaultCloudProviderType: aws
 
+templatesNeedingRoot: [ aws-chroot.json ]
```

Comment:
This value needs to be specified without the enclosing brackets:

`templatesNeedingRoot: aws-chroot.json`

It is not a proper yaml list, just a string representing a list of comma-separated values that the spring conversion stuff will tokenize.